src/OpenTelemetry.Api/Trace/SpanHelper.cs

```diff
@@ -24,7 +24,9 @@ namespace OpenTelemetry.Trace
     /// </summary>
     public static class SpanHelper
     {
+#pragma warning disable CA1805 // Do not initialize unnecessarily
```

Comment:
We don't we use `default` directly instead of `DefaultStatus`?

Reply:
In the best I tried to add this change. And we saw that it was done that way because of perf. Doing benchmark with and without the numbers go up a little. I will try to find out where is are those.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/pull/1016#discussion_r466722640